### ISSUE TITLE
[Accessibility] In high contrast mode the dropdown icons are not visible

### ISSUE THREAD

**Issue by jiaqi3** [ng-select/ng-select#2139]
**To Reproduce** 
Steps to reproduce the behavior (if example is not provided):
chose a hign constrast mode in display settings.

**Expected behavior**
show the right icon.

**Screenshots**
collapsed:
![image](https://user-images.githubusercontent.com/70375028/191936767-e9933f38-f2ab-4f2d-a421-3a7e9a8fbda1.png)
expended:
![image](https://user-images.githubusercontent.com/70375028/191936857-e5f94db6-3022-47a8-ae25-73017839eebd.png)
